api/src/opentrons/protocol_engine/commands/load_liquid.py

```
@@ -0,0 +1,88 @@
+from __future__ import annotations
+from pydantic import BaseModel, Field
+from typing import Optional, Type, Dict, TYPE_CHECKING
+from typing_extensions import Literal
+
+from .command import AbstractCommandImpl, BaseCommand, BaseCommandCreate
+from ..errors import (
+    LiquidNotFoundError,
+    LabwareNotLoadedError,
+    WellDoesNotExistError,
+)
+
+if TYPE_CHECKING:
+    from ..state import StateView
+
+LoadLiquidCommandType = Literal["loadLiquid"]
+
+
+class LoadLiquidParams(BaseModel):
+    """Payload required to load a liquid into a well."""
+
+    liquidId: str = Field(
+        ...,
+        description="Unique identifier of the liquid to load.",
+    )
+    labwareId: str = Field(
+        ...,
+        description="Unique identifier of labware to load liquid into.",
+    )
+    volumeByWell: Dict[str, float] = Field(
+        ...,
+        description="Volume of liquid, in µL, loaded into each well by name, in this labware.",
+    )
+
+
+class LoadLiquidResult(BaseModel):
+    """Result data from the execution of a LoadLiquid command."""
+
+    pass
+
+
+class LoadLiquidImplementation(AbstractCommandImpl[LoadLiquidParams, LoadLiquidResult]):
+    """Load liquid command implementation."""
+
+    def __init__(self, state_view: StateView, **kwargs: object) -> None:
+        self._state_view = state_view
+
+    async def execute(self, params: LoadLiquidParams) -> LoadLiquidResult:
+        """Load data necessary for a liquid."""
+        if not self._state_view.liquid.has(params.liquidId):
+            raise LiquidNotFoundError(
+                f"Supplied liquidId: {params.liquidId} does not exist in the loaded liquids."
+            )
+
+        try:
+            labware_wells = self._state_view.labware.get_wells(params.labwareId)
+        except KeyError:
+            raise LabwareNotLoadedError(
+                f"Supplied labwareId: {params.labwareId} does not exist in the loaded labware."
+            )
```

I would raise this error in the selector rather than put a try/except here, where it's harder to test